packages/contracts/contracts/interfaces/v0.8.x/ISharedMinterPolyptychV0.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title This interface adds support for polyptych mints when purchasing.
+ * @author Art Blocks Inc.
+ */
+interface ISharedMinterPolyptychV0 {
+    // Emitted when the polyptych panel ID of a project is updated.
+    event UpdatedPolyptychProjectPanelId(
```

Comment:
we didn't index this previously, but seems prudent to do so once this becomes a shared minter

Reply:
this was updated to use a generic event and is outdated